suite-common/message-system/config/config.v1.json

```diff
@@ -795,44 +795,23 @@
         {
             "conditions": [
                 {
-                    "settings": [
-                        {
-                            "zec": true
-                        }
-                    ],
                     "environment": {
-                        "desktop": ">=22.6.2",
+                        "desktop": ">=21.7.1",
                         "mobile": "!",
-                        "web": ">=22.6.2"
+                        "web": ">=21.7.1"
```

Comment:
why from 21.7.1 Suite version? Is it the version where FW 1.7.0 has been introduced?

Reply:
It is one of the first Suite released, so I wanted to make sure that practically everyone will receive it

Reply:
Then you can use `*` for any version or remove that environment condition completely.